packages/scanner/src/page.ts

```diff
@@ -40,27 +42,53 @@ export class Page {
         try {
             response = await gotoUrlPromise;
         } catch (err) {
-            return { error: `Puppeteer navigation to ${url} failed: ${(<Error>err).message}` };
+            this.logger.logError('url navigation failed', { scanError: JSON.stringify(err) });
+
+            //  `Puppeteer navigation to ${url} failed: ${(<Error>err).message}`
+            return { error: this.getScanErrorFromNavigationFailure((err as Error).message) };
         }
 
         if (!this.isHtmlPage(response)) {
-            return { unscannable: true, error: `Cannot scan ${url} because it is not a html page.` };
+            const contentType = this.getContentType(response.headers());
+
+            this.logger.logError('url is not a html page', { contentType: contentType });
```

Comment:
Neat - 'The URL returned non-HTML content'
Do we need to add URL in log params?